Makefile.am

```diff
@@ -63,13 +70,15 @@ libdill_la_LDFLAGS = -no-undefined -version-info @DILL_LIBTOOL_VERSION@
 libdill_la_CFLAGS = \
     -fvisibility=hidden\
     -DDILL_EXPORTS \
-    -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0
```

Comment:
Are you sure this can be removed? I've encountered cases where stack fortification crashed the process when in faced libdill stack trickery.